docs/examples/qemu_arm64.md

```diff
@@ -0,0 +1,135 @@
+# The arm64 images
+
+EB corbos Linux comes with arm64 baesedxample images for rpi4 and nxp s32g boards at the moment. To ease development and testing we also provide QEMU arm64 images.
+
+For _arm64/qemu_ we provide example images for EB corbos Linux (EBcL) and for Ubuntu Jammy. The difference between EBcl and Jammy is, that EBcL provides some additional components, like the _crinit_ init-manager and the _elos_ logging and event framework, and that EBcL provides a qualified security maintenance release every three months, while Jammy is proving updates continuously, using less strict qualification and documentation.
```

Comment:
Shall we change the documentation structure? This was already mentioned in the amd64 section.

Reply:
PRs to improve the docs are welcome ;)